test/configure.test.ts

```diff
@@ -1041,6 +1042,12 @@ suite("Configure (Add Docker files to Workspace)", function (this: Suite): void
     // Java
 
     suite("Java", () => {
+        if (os.platform() === 'win32') {
+            // Skip tests that are faulty on Windows
+            // TODO: re-enable?
```

Comment:
Do we need an issue to track this? otherwise we will forget this.

Reply:
The tests are sorely in need of some serious TLC. We should consider a much bigger test improvement task...

Reply:
I think we should add a task to revisit the test infrastructure, including whether we bother with some/all of the automated tests).

Reply:
#1737